packages/@aws-cdk/aws-ecr-assets/test/tarball-asset.test.ts

```diff
@@ -0,0 +1,150 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { expect as ourExpect, haveResource } from '@aws-cdk/assert-internal';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cxschema from '@aws-cdk/cloud-assembly-schema';
+import { App, Stack } from '@aws-cdk/core';
+import * as cxapi from '@aws-cdk/cx-api';
+import { testFutureBehavior } from 'cdk-build-tools/lib/feature-flag';
+import { TarballImageAsset } from '../lib';
+
+/* eslint-disable quote-props */
+
+const flags = { [cxapi.NEW_STYLE_STACK_SYNTHESIS_CONTEXT]: true };
+
+describe('image asset', () => {
+  testFutureBehavior('test instantiating Asset Image', flags, App, (app) => {
+    // GIVEN
+    const stack = new Stack(app);
+    const assset = new TarballImageAsset(stack, 'Image', {
+      tarballFile: 'test/demo-tarball/empty.tar',
```

Comment:
Use __dirname to make this relative to the current source file. Don't assume where the test is running from